itests/test_suite.sh

```diff
@@ -52,7 +52,7 @@ rm -rf ~/.jbang/cache
 echo Testing with `which jbang`
 
 ## init ##
-assert "jbang init $SCRATCH/test.java"
+assert "jbang --verbose init $SCRATCH/test.java"
```

Comment:
why this change?

Reply:
IMHO can be undone, I think that was done early in this PR, probably had some issue so needed debug logs for resolver?